### PR TITLE
Use log.error instead of log.external on update frequency logs

### PR DESCRIPTION
**Pull request by andrecardoso**

### CODE REVIEW

**Comment by andrecardoso** [comment]
[(1) Update Player error logs to use 'error'](https://trello.com/c/cpWjAFSh/4690-1-update-player-error-logs-to-use-error)

File: src/main/player/content-comparison.js [hunk 1]
```diff
@@ -91,6 +91,6 @@ function writeContentDates(items) {
   const filePath = path.join(commonConfig.getInstallDir(), contentComparisonFileName);
   return platform.writeTextFile(filePath, JSON.stringify(items, null, 2))
     .catch(err => {
-      log.external(`error updating ${contentComparisonFileName} contents`, util.inspect(err));
+      log.error(util.inspect(err), `error updating ${contentComparisonFileName} contents`);
```
Comment (rudibravo) [inline src/main/player/content-comparison.js]: @andrecardoso I think the message "error updating contents" should be enough for the event_details and we don't need the stack on this error.

Comment (andrecardoso) [reply src/main/player/content-comparison.js]: Agreed. This is a very specific error the stack doesn't add much. Thanks.

**Comment by andrecardoso** [comment]
@rudibravo can you please re-review? Thanks.

**Pull request merged**